com.ca.lsp.cobol/lsp-service-cobol/src/main/java/com/ca/lsp/cobol/service/utils/SettingsParametersEnum.java

```diff
@@ -20,7 +20,9 @@
  */
 public enum SettingsParametersEnum {
   CPY_MANAGER("cpy-manager"),
-  LSP_PREFIX("broadcom-cobol-lsp");
+  LSP_PREFIX("broadcom-cobol-lsp"),
+  LOCAL_PATHS("cpy-manager.paths.local"),
+  DATASETS("cpy-manager.paths.dsn");
```

Comment:
Is DATASETS used?  If yes, we can still alphabetize.

Reply:
We don't use it on the server, but it is a value that we support on the client. I would prefer to leave it here to keep consistency.

Reply:
Then let's alphabetize the enum-list